### PR TITLE
Add-Muthu-Annamalai to README.MD

### PR DESCRIPTION
**Pull request by muthuannamalai12**
Updated README.md and added a new file called code of conduct for our project to show to our contributors in README.md

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see
```
Comment (SanjayMarreddi) [inline CODE_OF_CONDUCT.md]: For answers to common questions about this code of conduct, see [this](https://www.contributor-covenant.org/faq).

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
```
Comment (SanjayMarreddi) [inline CODE_OF_CONDUCT.md]: *Avoid Keeping Links directly.*
This Code of Conduct is adapted from the ...   is available [here](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html).

File: README.md [hunk 3]
```diff
@@ -1,27 +1,35 @@
...
-# :sunglasses: Facial Expression Recognition Classifier Model 
+## :sunglasses:  About The Project:
+
+# Facial Expression Recognition Classifier Model
 
-A Facial Expression Recognition Classifier Model that takes
-**Real-time Video input** and predicts the **Emotion of users** present in front of
-Webcam. It also gives **Graphical Visualization** of Expressions when we feed in an
-Image via Web Cam or manually !
+Facial expression for emotion detection has always been an easy task for humans, but achieving the same task with a computer algorithm is quite challenging. With the recent advancement in computer vision and machine learning, it is possible to detect emotions from images.In this project,we propose a novel technique called facial emotion recognition using convolutional neural networks,python and flask. Facial expressions are the vital identifiers for human feelings, because it corresponds to the emotions. Most of the times (roughly in 55% cases) [1], the facial expression is a nonverbal way of emotional expression, and it can be considered as concrete evidence to uncover whether an individual is speaking the truth or not [2].
```
Comment (SanjayMarreddi) [inline README.md]: Remove [1] and [2] Numbering in the Last Statement

**Review by SanjayMarreddi — CHANGES_REQUESTED** [review]
Good Work. 
Look at the Comments below & modify accordingly so that I can merge it.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (muthuannamalai12) [reply CODE_OF_CONDUCT.md]: I have Changed it

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (muthuannamalai12) [reply CODE_OF_CONDUCT.md]: Ok. I have changed it Thank you

File: README.md (hunk 3 above)
Comment (muthuannamalai12) [reply README.md]: I have changed it

**Comment by muthuannamalai12** [comment]
@SanjayMarreddi Can you please add DWOC label and the level label on my pull request

**Comment by SanjayMarreddi** [comment]
@muthuannamalai12 Done !

**Comment by muthuannamalai12** [comment]
> @muthuannamalai12 Done !

Thank you

**Pull request merged**